packages/downloads/src/hub/hub-request-dataset-export.ts

```diff
@@ -0,0 +1,74 @@
+import { RemoteServerError } from '../remote-server-error';
+import { composeDownloadId } from '../utils';
+import { DownloadFormat } from "../download-format";
+import { convertToHubFormat } from "./format-converter";
+
+export interface IHubDatasetExportRequestParams {
+  host: string;
+  datasetId: string;
+  format: DownloadFormat;
+  spatialRefId?: string;
+  spatialRefWkt?: string;
```

Comment:
Unclear what `spatialRefWkt` is for as it does not seem to be used in the fn below...

Reply:
Ah, we were talking about supporting `spatialRefWkt` in the future, and the first draft of the code had it in there.  Since the current hub API doesn't support I decide to remove, but missed this one. Will remove.

Reply:
👍